exporter/kafkaexporter/README.md

```diff
@@ -32,7 +32,7 @@ The following settings can be optionally configured:
     - `key_file`: path to the TLS key to use for TLS required connections. Should
       only be used if `insecure` is set to true.
     - `insecure` (default = false): Disable verifying the server's certificate chain and host 
-      name (`InsecureSkipVerify` in the tls config)
+      name (`insecure_skip_verify` in the tls config)
```

Comment:
I find the description of this flag still somewhat confusing. Basing on the [comment in configtls.go](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.45.0/config/configtls/configtls.go#L64-L69) I understand that for HTTP, `insecure` acts in a similar way as `insecure_skip_verify` for gRPC. I don't think changing this to snake case (like in the yaml) changes much in making this more clear. Perhaps it could instead just be skipped altogether or more context could be provided, e.g. (not ideal either, but just wanted to share one idea how this could be phrased): 

```suggestion
      name (equivalent to [`insecure_skip_verify`](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.45.0/config/configtls/configtls.go#L72) in the tls config since it is [used for HTTP](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.45.0/config/configtls/configtls.go#L64-L69))
```